.github/workflows/terraform-daily-healthcheck-deploy-and-destroy.yml

```diff
@@ -1,90 +1,183 @@
-name: 'Daily Healthcheck Deploy and Destroy'
+name: 'Daily Healthcheck Deploy, Test and Destroy'
 
-#on:
-#  schedule:
-#    - cron: 45 7 * * 1-5
 on:
-  workflow_dispatch:
-    inputs:
-      buildBranch:
-        description: 'Feature branch to push to sandbox.'
-        required: true
-        type: 'string'
-      sandboxWorkspace:
-        description: 'Which Sandbox to push to.'
-        required: true
-        type: 'string'
-      environment:
-        default: 'development'
-        description: 'Which environment should this run against'
-        required: true
-        type: 'string'
-#
-#permissions:
-#  pull-requests: write
-#  id-token: write # This is required for requesting the JWT
-#  contents: read # This is required for actions/checkout
+ schedule:
+   - cron: 30 4 * * 1-5
+
+permissions:
+  id-token: write 
+  contents: read
 
 jobs:
-  test-job:
+  terraform_process:
     runs-on: ubuntu-latest
+    environment: development
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: main
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE }}
+          role-skip-session-tagging: true
+          aws-region: ${{ vars.AWS_REGION }}
+          mask-aws-account-id: true
+
+      - name: View AWS Role
+        run: aws sts get-caller-identity
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v3
+        with:
+          terraform_version: 1.5.4
+          terraform_wrapper: false
+
+      - name: Terraform Init
+        id: init
+        run: terraform init -backend-config=backend.conf
+        working-directory: ./infrastructure
+        shell: bash
+
+      - name: Terraform Set Workspace
+        id: workspace
+        run: terraform workspace select -or-create ndrd
```

Comment:
Any chance we can put `ndrd` at the top as an `env` var called `workspace`? eg
```
env:  
  WORKSPACE: ndrd
  
- name: Terraform Set Workspace
        id: workspace
        run: terraform workspace select -or-create $WORKSPACE
```

Just so it's clear exactly what and where to set the workspace if this were to ever change in future